fastapi/openapi/models.py

```diff
@@ -400,6 +400,14 @@ class OpenAPI(BaseModel):
     class Config:
         extra = "allow"
 
+    @validator("tags")
+    def check_foo(cls, v):  # type: ignore
```

Comment:
let's add descriptive names? Ref: https://google.github.io/styleguide/pyguide.html#316-naming

Reply:
Sorry about that, that was a lazy copy-paste error from the pydantic docs :) fixed